Project.toml

```diff
@@ -1,20 +1,13 @@
 name = "QEDevents"
 uuid = "fc3ce04a-5be5-4f3a-acff-eceaab723759"
-authors = [
-    "Uwe Hernandez Acosta <u.hernandez@hzdr.de>",
-    "Simeon Ehrig",
-    "Klaus Steiniger",
-    "Tom Jungnickel",
-    "Anton Reinhard",
-]
+authors = ["Uwe Hernandez Acosta <u.hernandez@hzdr.de>", "Simeon Ehrig", "Klaus Steiniger", "Tom Jungnickel", "Anton Reinhard"]
```

Comment:
Again? Let's see who wins this battle 🥷

Reply:
Maybe we should figure out a way to have this consistently formatted 🙃